lib/Dancer2/Plugin/UnicodeNormalize.pm

```diff
@@ -6,18 +6,17 @@ package Dancer2::Plugin::UnicodeNormalize;
     $Dancer2::Plugin::UnicodeNormalize::VERSION = '0.033';
 }
 
-use Dancer2 0.14300;
 use Dancer2::Plugin;
 use Unicode::Normalize;
 
 on_plugin_import {
     my $dsl = shift;
-    my $settings = plugin_setting();
 
     $dsl->app->add_hook(
         Dancer2::Core::Hook->new(
             name => 'before',
             code => sub {
+                my $settings = plugin_setting;
```

Comment:
@jbarrett sucky but effective workaround supplied as PR #5 which also resolves the request deprecation warnings.